backend/app/services/rag/pipeline/docling.py

```
@@ -0,0 +1,325 @@
+            docling_url: Docling service URL endpoint
+            chunk_size: Maximum chunk size in characters
+            chunk_overlap: Number of characters to overlap between chunks
+            timeout: HTTP request timeout in seconds
+
+        Raises:
+            ValueError: If docling_url is empty
+        """
+        super().__init__(chunk_size, chunk_overlap)
+
+        if not docling_url:
+            raise ValueError("Docling URL is required")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)